src/mongo/db/curop.h

```diff
@@ -527,6 +535,9 @@ class CurOp {
 
     std::string _planSummary;
 
+    // Pseudo RNG for rate limiter feature
+    static PseudoRandom _prng;
```

Comment:
Ok, agree,  I can remove this from the header. But in your original comment you mentioned creating additional method. I did not understand that part.